dts/arm/nxp/nxp_lpc55S6x_common.dtsi

```diff
@@ -192,8 +192,10 @@
 
 	hs_lspi: spi@9f000 {
 		compatible = "nxp,lpc-spi";
-		cs-gpios = <&gpio0 20 0>,<&gpio1 1 0>,<&gpio1 12 0>,
-			<&gpio1 26 0>;
+		/* The comment below will allow using GPIO CS,
+		 rather than Flexcomm SS*/
+		/* cs-gpios = <&gpio0 20 0>,<&gpio1 1 0>,<&gpio1 12 0>,
```

Comment:
Are these GPIOs defined by the SoC for CS purposes?

Reply:
@galak Yes, these pins are mapped on SoC to be used as spi slave select. This means they are wired as CS for SPI devices on the evaluation boards.

Reply:
Comment about GPIO CS needs to be fixed after #26269 was merged, e.g. `/* cs-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>, ....*/`

Reply:
@jfischer-phytec-iot i've updated the dts file